tests/InputFiles/ScalarWave/ObserveExample.yaml

```diff
@@ -0,0 +1,53 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: EvolveScalarWave1D
+
```

Comment:
Add `# Check:` for the test to work (at least that's what `AddInputFileTests.cmake` says)

Or, add `# Check: execute` and `# ExpectedOutput:` (see `Poisson/Input1D.yaml` for an example) so that it is tested that files are actually written to disk.

Reply:
Looks like it works to me (from Travis):
```
 29/948 Test  #29: InputFiles.ScalarWave.ObserveExample.yaml.parse ............................   Passed    0.12 sec
```